great_expectations/core/batch_spec.py

```diff
@@ -107,3 +108,40 @@ def query(self):
     @query.setter
     def query(self, query) -> None:
         self["query"] = query
+
+
+class GlueDataCatalogBatchSpec(BatchSpec):
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        if "database_name" not in self:
+            raise InvalidBatchSpecError(
+                "GlueDataCatalogBatchSpec requires a database_name"
+            )
+        if "table_name" not in self:
+            raise InvalidBatchSpecError(
+                "GlueDataCatalogBatchSpec requires a table_name"
+            )
+
+    @property
+    def reader_method(self) -> dict:
+        return "table"
+
+    @property
+    def database_name(self) -> str:
+        return self.get("database_name")
+
+    @property
+    def table_name(self) -> str:
+        return self.get("table_name")
+
+    @property
+    def path(self) -> str:
+        return f"{self.database_name}.{self.table_name}"
+
+    @property
+    def reader_options(self) -> dict:
+        return self.get("reader_options", {})
+
+    @property
+    def partitions(self) -> List[str]:
```

Comment:
@lccasagrande I am not seeing the `partitions` property set.  Please point me to it (sorry).  If not meant to be set ever, then I am not sure why the getter is there.  Please elaborate, if you do not mind.  Thank you!